webpack.config.js

```diff
@@ -38,6 +38,18 @@ module.exports = {
         test: /\.(woff|woff2|eot|ttf|otf)$/i,
         type: 'asset/resource',
       },
+      {
+        test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+        include: path.resolve(__dirname, './node_modules/bootstrap-icons/font/fonts'),
+        use: {
+          loader: 'file-loader',
+          options: {
+            name: '[name].[ext]',
+            outputPath: 'webfonts',
+            publicPath: '../webfonts',
+          },
+        },
+      },
```

Comment:
Is this configuration block necessary if we are not using Bootstrap? If Bootstrap is not a part of our project, we may consider removing it to keep a lighter configuration focused on our specific needs.